features/step_definitions/us13_borrowers_personal_info.rb

```diff
@@ -70,6 +70,7 @@
     title_number: 'DT567568',
     md_ref: 'e-MD12344',
     identity_checked: 'Y',
+    address: '1 High Street, High Wycombe, CR0 6TY',
     borrowers: [
       {
         forename: 'Katie',
```

Comment:
us40_add_md_ref.rb also has a step which manually sets the @deed_hash object, even though this step is supposed to fail it's probably worth updating it with the address to make sure it is failing for the purpose of it's test no-mdref rather than because the address isnt there.